kubernetes/src/kubernetes/watch.rb

```diff
@@ -23,9 +23,16 @@ def initialize(client)
       @client = client
     end
 
-    def connect(path, &_block)
+    def make_url(path, resource_version)
+      query = '?watch=true'
+      query += "&resourceVersion=#{resource_version}" if resource_version
+      path + query
+    end
+
+    def connect(path, resource_version, &_block)
```

Comment:
Maybe its worth setting this as `resource_version = nil` so it isn't a breaking change?

Reply:
done. (learn something new about Ruby every day :)